internal/broker/testdata/TestConcurrentIsAuthenticated/golden/first_auth_starts_and_finishes_before_second/second_auth

```diff
@@ -1,3 +1,3 @@
 access: granted
-data: '{"userinfo":{"name":"user-timeout-1","uuid":"saved-user-id","dir":"/home/user-timeout-1","shell":"/usr/bin/bash","gecos":"user-timeout-1","groups":[{"name":"remote-group","ugid":"12345"},{"name":"linux-local-group","ugid":""}]}}'
+data: '{"userinfo":{"name":"test-user@email.com","uuid":"saved-user-id","dir":"/home/test-user@email.com","shell":"/usr/bin/bash","gecos":"test-user@email.com","groups":[{"name":"saved-remote-group","ugid":"12345"},{"name":"saved-local-group","ugid":""}]}}'
```

Comment:
Although it might be following the right order in the code, the golden files do not mirror the expected behavior (i.e. we are getting the same user info for both calls, even though we used different usernames), so we need to try and make it resemble more the real scenario.

Reply:
I'm not sure how this happened, but it seems like your comment is made on an outdated diff. On commit 7cfda81 (pushed two weeks ago), I see the following diff (which contains different usernames in the response of the two calls):

```diff
diff --git a/internal/broker/testdata/TestConcurrentIsAuthenticated/golden/first_auth_starts_and_finishes_before_second/first_auth b/internal/broker/testdata/TestConcurrentIsAuthenticated/golden/first_auth_starts_and_finishes_before_second/first_auth
index bd6241b..13dbd87 100644
--- a/internal/broker/testdata/TestConcurrentIsAuthenticated/golden/first_auth_starts_and_finishes_before_second/first_auth
+++ b/internal/broker/testdata/TestConcurrentIsAuthenticated/golden/first_auth_starts_and_finishes_before_second/first_auth
@@ -1,3 +1,3 @@
 access: granted
-data: '{"userinfo":{"name":"user-timeout-0","uuid":"saved-user-id","dir":"/home/user-timeout-0","shell":"/usr/bin/bash","gecos":"user-timeout-0","groups":[{"name":"remote-group","ugid":"12345"},{"name":"linux-local-group","ugid":""}]}}'
+data: '{"userinfo":{"name":"user1","uuid":"saved-user-id","dir":"/home/user1","shell":"/usr/bin/bash","gecos":"user1","groups":[{"name":"new-broker-for-tests-remote-group","ugid":"12345"},{"name":"linux-new-broker-for-tests-local-group","ugid":""}]}}'
 err: <nil>
diff --git a/internal/broker/testdata/TestConcurrentIsAuthenticated/golden/first_auth_starts_and_finishes_before_second/second_auth b/internal/broker/testdata/TestConcurrentIsAuthenticated/golden/first_auth_starts_and_finishes_before_second/second_auth
index 1fee6db..c0b252f 100644
--- a/internal/broker/testdata/TestConcurrentIsAuthenticated/golden/first_auth_starts_and_finishes_before_second/second_auth
+++ b/internal/broker/testdata/TestConcurrentIsAuthenticated/golden/first_auth_starts_and_finishes_before_second/second_auth
@@ -1,3 +1,3 @@
 access: granted
-data: '{"userinfo":{"name":"user-timeout-1","uuid":"saved-user-id","dir":"/home/user-timeout-1","shell":"/usr/bin/bash","gecos":"user-timeout-1","groups":[{"name":"remote-group","ugid":"12345"},{"name":"linux-local-group","ugid":""}]}}'
+data: '{"userinfo":{"name":"user2","uuid":"saved-user-id","dir":"/home/user2","shell":"/usr/bin/bash","gecos":"user2","groups":[{"name":"new-broker-for-tests-remote-group","ugid":"12345"},{"name":"linux-new-broker-for-tests-local-group","ugid":""}]}}'
 err: <nil>
```

Reply:
Yeah, it was outdated indeed. Not sure why I was seeing the outdated version, but now I can see that it's correct.